apps/base/utils.py

```diff
@@ -168,6 +168,28 @@ def get_boto3_client(resource, aws_keys):
         logger.exception(e)
 
 
+def get_presigned_url_for_file_upload(filename, file_key):
+    """
+    """
+    try:
+        aws_keys = {
```

Comment:
@KhalidRmb aren't these loaded in settings? can't we use those? If not can you create a method in utils and move this piece of code there so that we avoid redundancy

Reply:
I've changed the values to those from settings instead of using env variables.